jfxui/src/uiTest/java/org/itsallcode/whiterabbit/jfxui/JavaFxAppUiTest.java

```diff
@@ -225,23 +226,25 @@ void selectedDayUpdatedWhenDayChanges()
     }
 
     @Test
+    @Tag(JunitTags.FLAKY)
     void weekendsAreHighlightedAsWeekend()
     {
         final JavaFxTable<DayRecordPropertyAdapter> dayTable = app().genericDayTable();
         assertAll(
-                () -> dayTable.assertRowHasPseudoClass(0, "weekend"),
-                () -> dayTable.assertRowHasPseudoClass(1, "weekend"),
-                () -> dayTable.assertRowDoesNotHavePseudoClass(2, "weekend"));
+                () -> dayTable.assertRowHasPseudoClass(7, "weekend"),
```

Comment:
This test is hard to judge if you don't document what the magic numbers are.

Reply:
I will refactor the UI-Tests in a separate PR, OK?